sphinx-doc/credits.rst

```diff
@@ -95,3 +95,868 @@ The following people have contributed to the to HOOMD-blue:
 * Wouter Ellenbroek, Eindhoven University of Technology
 * Yuan Zhou, University of Michigan
 * Åsmund Ervik, SINTEF
+
+=======
```

Comment:
Why are these changes here? Are they just hold outs from a removal in master?

Reply:
This looks like a merge conflict that was not resolved: it starts with `=======`

Reply:
It is a merge conflict that I failed to resolve correctly. Do I remove everything from line 3 to "====="  on line 99?

Reply:
I would recommend just replacing the credits.rst file with the one from master. Then add your names to it.